### ISSUE TITLE
Show copyright page in web version

### ISSUE THREAD

**Issue by programmieraffe** [pressbooks/pressbooks#108]
The copyright info text (book info) is not shown with full details in the web version.

**Comment by hughmcguire** [comment]
do you think this should be in the footer of every page / chapter of the web version of the book?

**Comment by programmieraffe** [comment]
Sorry for the late answer. I opened this enhancement, because I think that the **Copyright Notice** (from Book information) should be shown as a single book page at the beginning of each book in the web version as it is already realized in the export versions (epub2/3). 

Reason 1: If I have critical license information in the copyright notice field it is nowhere shown in the web version.

Additionally I think the copyright section needs some improvement, because the Creative Commons license info does not contain the specific version of the license and no link to the license text which is required. So two fields should be added:
- license title
- url to link text
That would be an advantage, because other licences can be used. 

Hope I clarified this.

**Comment by bdolor** [comment]
We have found it necessary to put the [**Copyright Notice** on the home/book page](https://github.com/BCcampus/pressbooks-textbook/blob/dev/themes-book/opentextbook/page-cover-bottom-block.php#L13)  for the open textbook theme.

In fact, the **Copyright License** section where you get to choose your license *does* include a link to the license text in two places so long as you choose to **Display the copyright license** in Appearance->Theme Options. Both the image and the license text are linked back to the specific license details on the creative commons website. 

![screen shot 2015-08-11 at 9 03 08 am](https://cloud.githubusercontent.com/assets/2048170/9202676/d59c7896-4007-11e5-88c7-e385600d305d.png)

The use of the CC API to generate the license attribution statement assures me that everything that is required by CC is provided by the API, [namely TASL (Title, Author, Source, Link)](https://wiki.creativecommons.org/wiki/Best_practices_for_attribution) It does not include a version of the license simply because the CC API doesn't provide a version option, and after speaking with the folks at CC we understand that version numbers don't need to be maintained throughout the life of a book. It is perfectly acceptable to update the version number of the license. We have books that, when first created were released under a 2.5 license and we have been instructed by them that nothing is lost when you update the license version (4.0) to the latest.
